qa/L0_grpc/test.sh

```diff
@@ -82,8 +83,11 @@ if [[ "$(< /proc/sys/kernel/osrelease)" == *microsoft* ]]; then
     SIMPLE_IMAGE_CLIENT=${SDKDIR}/python/image_client
     # SIMPLE_ENSEMBLE_IMAGE_CLIENT=${SDKDIR}/python/ensemble_image_client
     SIMPLE_REUSE_INFER_OBJECTS_CLIENT=${SDKDIR}/python/reuse_infer_objects_client
+    # [FIXME] point to proper client
```

Comment:
Do these FIXME's need changes?

Reply:
Not in this change, this should be taken care of when we port this test to Windows platform as well, note that the paths for the other clients above also need to be fixed ("python" path for cpp clients)